lib/bindings/python/rust/lib.rs

```diff
@@ -348,7 +359,7 @@ impl EtcdClient {
 impl Client {
     /// Get list of current endpoints
     fn endpoint_ids(&self) -> Vec<i64> {
-        self.inner.endpoint_ids().borrow().clone()
+        self.inner.endpoint_ids()
```

Comment:
Why did we have §.borrow.clone()§ before?

Reply:
That now moved inside `endpoint_ids()`